tests/unit/amo/test_withExperiment.js

```diff
@@ -415,7 +423,8 @@ describe(__filename, () => {
   it('does not have any invalid experiment ids defined in the config', () => {
     // If this test fails it is because an experimentId does not match the
     // expected format of YYYYMMDD_ExperimentName.
-    for (const experimentId of Object.keys(config.get('experiments'))) {
+    // eslint-disable-next-line import/namespace
```

Comment:
mm, again?

Reply:
The message from eslint is `'experiments' not found in imported namespace 'defaultConfig'`

Reply:
ah ok, I see now, thanks